src/csi/deploy/kubernetes/storageclass.yaml

```diff
@@ -7,9 +7,9 @@ provisioner: csi.pos.io
 parameters:
   fsType: ext4
   transportType: "tcp"
-  targetAddress: "107.108.83.97"
+  targetAddress: "107.99.41.213"
```

Comment:
If localhost or 127.0.0.1 works than use it else we can mention to put IP.

Reply:
It's not desirable to expose the internal IP address. An (mandatory) alternative is to replace it with an environmental variable. It's always not good to exposure the IP this way.

Reply:
Yes, Thanks for pointing out, I will remove the IP address and keep localhost for time being
and we will think about to environmental variable approach in CSI deployment